### DISCUSSION TITLE
[NASA-Question] Address for Filter Reset

### DISCUSSION THREAD

**Discussion by snzyeee** [omerfaruk-aran/esphome_samsung_hvac_bus#247] [Questions]
### Your Question

Just curious if anyone found the address to reset the Clean Filter time. I couldnt find it yet.

Theres this lovely "myehs" Protocol Wiki, but sadly i couldnt find the message for "NASA_FILTER_CLEAN".

### Pre-Submission Checklist

- [x ] I have searched for a similar question in open/closed issues.
- [x ] I have checked the project’s documentation for relevant information.
- [x ] I have provided as much detail as possible to help answer my question.

### HVAC System Type

None

### Samsung HVAC Device Model

Samsung Windfree Avant

### ESP Device Model

M5 Atom

**Comment by killtux** [comment]
Yeah it would be really nice to get the "Reset Dust Filter" Button integrated. 
It's the only thing thats prevting me from ditching the Smart Things Integration.

![image](https://github.com/user-attachments/assets/826c0c15-8ba4-4881-827a-3c73a0268494)

**Comment by bluewalk** [comment]
Any update on or plans for this? I had to search for my remote as CF popped up 😅

**Comment by trianglesis** [comment]
To bring this UP!

I've added some absent functions as IR buttons for the "Athom IR 2" mod in ESP Home.
So there is a way to control the Beep, Filter reset or Display Off functions for my AC unit.

However, will be nice to have buttons in the NASA protocol/module.

**Reply by killtux** [reply]
> So there is a way to control the Beep, Filter reset or Display Off functions for my AC unit.
> However, will be nice to have buttons in the NASA protocol/module.

this would be awesome!
